src/core/render/index.js

```diff
@@ -294,7 +294,8 @@ export function renderMixin(proto) {
   };
 
   proto._renderNav = function(text) {
-    text && this._renderTo('nav', this.compiler.compile(text));
+    text &&
+      this._renderTo(this.config.nav_el || 'nav', this.compiler.compile(text));
     if (this.config.loadNavbar) {
       getAndActive(this.router, 'nav');
```

Comment:
- [ ] This relies on the `'nav'` selector, and should be either (1) updated to account for `nav_el`, or (2) disabled if `nav_el` is defined, or (3) maybe add a new option like `showActiveNavbarLinks` to be able to configure separately from `nav_el` while using `nav_el`.